### PR TITLE
strip catkin-sphinx undeclared dependency

### PR DESCRIPTION
**Pull request by tfoote**
It was undeclared and this makes it completely self contained.

### CODE REVIEW

**Comment by wjwwood** [comment]
Did you just copy the contents of the `catkin-sphinx` package into here?

**Comment by tfoote** [comment]
@wjwwood this is the theme from catkin-sphinx. Please see https://github.com/ros-infrastructure/catkin-sphinx/issues/6

**Comment by dirk-thomas** [comment]
From my point of view a `-1` on copy-n-pasting this into multiple repositories. If it is being used in more then one place it should stay separately.

An alternative option to consider would be to remove the custom theme altogether and just use the default.

**Comment by tfoote** [comment]
I would be fine with using the default theme too. I just did this to keep
exactly the same behavior. ​

**Comment by tfoote** [comment]
Using default theme

File: doc/conf.py [hunk 1]
```diff
@@ -104,8 +103,8 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme_path = [os.path.join(os.path.dirname(catkin_sphinx.__file__), 'theme')]
-html_theme = 'ros-theme'
+# html_theme_path = [os.path.join(os.path.dirname(__file__), 'theme')]
+# html_theme = 'ros-theme'
```
Comment (dirk-thomas) [inline doc/conf.py]: Please remove obsolete lines instead of commenting them out.

**Comment by dirk-thomas** [comment]
Can you please compare the documentation before and after this patch to ensure that all content is available in the same way.

**Comment by wjwwood** [comment]
-1 to duplicating, -0 to use the default. I think the custom theme is nice to have and I'm willing to spend time preserve it. Please tell what needs to be fixed and I'll make sure it's working.

File: doc/conf.py (hunk 1 above)
Comment (tfoote) [inline doc/conf.py]: I left them commented out to be consistent with the rest of the file. This is a standard template, every other customizable value is commented out throughout the file to show examples of how to do it.

Comment (dirk-thomas) [inline doc/conf.py]: I doubt that `ros-theme` is standard though.

**Pull request closed without merging**